pyproject.toml

```diff
@@ -53,6 +53,18 @@ optional = true
 [tool.poetry.group.postgres.dependencies]
 psycopg2 = "2.9.9"
 
+[tool.poetry.group.postgreslocal]
+optional = true
+
+[tool.poetry.group.postgreslocal.dependencies]
+psycopg2 = "2.9.9"
+django-template-check = "0.4.0"
+django-extensions = "^3.2.1"
+libsass = "0.22.0"
```

Comment:
Using this setup it also does not use the static css files generated during build for some reason, which is why I added libsass as a dep, but it should be doable without libsass like in prod

Reply:
I think the problem was in DJANGO_ENV=development. That tends to change static files and templates discovery